app/Console/Commands/MigrateSettingsData.php

```diff
@@ -39,6 +40,11 @@ class MigrateSettingsData extends Command implements Isolatable
             'handler' => 'schemalessPromptsMigration',
             'description' => 'Moves user, regulated organization and organizations’ prompts to the new prompts schemaless attributes column.',
         ],
+        'UpdateBlindIndexes' => [
```

Comment:
Now that we have this I think we should maybe change this console command to be a bit more generic, e.g. `MigrateData`, as it is not just dealing with settings anymore.

Reply:
I'm going to file a new issue for this.